### PR TITLE
Adding python-email

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by timkpaine** [comment]
@lavr do you want to be a maintainer on this?

File: recipes/email/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - chardet
+    - cssutils
+    - lxml
+    - premailer
+    - python-dateutil
+    - requests
+
+test:
+  imports:
+    - emails
```
Comment (synapticarbors) [inline recipes/email/meta.yaml:32]: Add imports of submodules (e.g. `emails.store`, `emails.backend`, etc)

Comment (timkpaine) [reply recipes/email/meta.yaml:32]: why?

Comment (timkpaine) [reply recipes/email/meta.yaml:32]: the author specifies what the top level exposes, who am I to assume the validity of submodule imports https://github.com/lavr/python-emails/blob/master/emails/__init__.py#L48

Comment (timkpaine) [reply recipes/email/meta.yaml:32]: or are you just trying to get it to match the setup.py? https://github.com/lavr/python-emails/blob/master/setup.py#L116

Comment (synapticarbors) [reply recipes/email/meta.yaml:32]: Part of it is matching the `setup.py`, part of it is just to extend the check to make sure that there aren't missing dependencies that aren't imported in a limited top level `__init__.py`

Comment (timkpaine) [reply recipes/email/meta.yaml:32]: ill add them, but I haven't done this for any other libraries recently

**Comment by timkpaine** [comment]
@synapticarbors ive omitted some that look like they're deliberately done (e.g. the Django submodule isn't imported at the top level, and Django isn't listed as a dependency, which seems deliberate)

File: recipes/email/meta.yaml (hunk 1 above)
Comment (synapticarbors) [reply recipes/email/meta.yaml:32]: I guess the reviewers aren't uniform in what they are looking for. I tend to check for that sort of thing, but some don't.

Comment (timkpaine) [reply recipes/email/meta.yaml:32]: no worries, easy to add.

**Pull request merged**